subsys/sd/sd.c

```diff
@@ -94,14 +94,8 @@ static int sd_common_init(struct sd_card *card)
 {
 	int ret;
 
-	/* Reset card with CMD0 */
-	ret = sd_idle(card);
```

Comment:
why can't you just use sd_retry to retry sending the cmd0?

Reply:
Because the whole purpose of CMD0 here is to make the card respond to CMD8. There is no way to tell if CMD0 has worked independently of another command

Reply:
I guess what I mean is like can you make a function that does CMD0 and CMD8 and call that with the sd_retry rather than changing this function signature for a specific case

Reply:
That makes much more sense, updated PR